OpenSim/Common/Component.h

```diff
@@ -907,6 +957,11 @@ OpenSim_DECLARE_ABSTRACT_OBJECT(Component, Object);
     /**
     * Get the Output provided by this Component by name.
     *
+    * <b>C++ example:</b> get an Output from a Component in a model
+    * @code{.cpp}
+    * model.getComponent("/path/to/component").getOutput("outputName");
+    * @endcode
+    *
     * @param name   the name of the cache variable
```

Comment:
Others may have a different opinion on this (since the comment is technically correct if all Outputs are cached), but I think the documentation should say that the argument to a method called "getOutput" is the name of the "Output". Regardless, the PR looks good and this comment can be ignored.